terraform/vars/terraform-staging.tfvars

```diff
@@ -10,6 +10,7 @@ suppress_no_config_warning= "true"
 control_tower_url         = "https://staging-api.resourcewatch.org"
 areas_api_url             = "https://gfw-staging.globalforestwatch.org/v1"
 geostore_api_url          = "https://gfw-staging.globalforestwatch.org/v1"
+rw_areas_api_url          = "https://api.resourcewatch.org/v2"
```

Comment:
Can you switch this to: `https://staging-api.resourcewatch.org/v2`

Can't mix and match between staging and production URL as the authentication will fail!